src/utility/verification/verify-resource.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import { Probot } from 'probot';
+import { Resource } from '../../service/resource/resource';
+
+export async function verifyOrgRepo(app: Probot, context: any, resource: Resource): Promise<boolean> {
+  const contextOrgName = context.payload.organization?.login || context.payload.repository?.owner?.login;
+  const contextRepoName = context.payload.repository?.name;
+
+  const org = resource.organizations.get(contextOrgName);
+  const repo = org?.repositories.get(contextRepoName);
+
+  if (!org || !repo) {
+    app.log.error(`${contextOrgName}/${contextRepoName} is not defined in resource config!`);
```

Comment:
So @peterzhuamazon we need to define the set of repos and org in the RESOURCE_CONFIG?

Reply:
Yes, that is exactly how we know if the call is made within the resource boundry we define.
Instead of any arbitrary repo can trigger the call.